Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/) for Kodi add-ons: skins/skin.estuary.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-add-ons-skins/-/skin-estuary/horizontal-auto.svg)

gade01 commented:
Everything looks good.

Mainly fixes done by me:
- Double spaces
- Trailing spaces
- Mismatching line breaks
- Mismatching brackets
- Removal of unchanged translations

ksooo commented on addons/skin.estuary/language/resource.language.de_de/strings.po:
```diff
@@ -186,7 +187,7 @@ msgstr "Wiedergabelisten-Einstellungen"
 msgctxt "#31043"
 msgid "Set the type and add rules to create a smart playlist. These playlists are dynamic and include all media items from your database which apply to your chosen rules."
-msgstr "Typ einstellen und Regeln für die intelligente Wiedergabeliste hinzufügen. Diese Art von Wiedergabeliste ist dynamisch und beinhaltet alle Medien aus der Bibliothek, die die gesetzten Regeln erfüllen."
+msgstr "Typ einstellen und Regeln für die intelligente Wiedergabeliste hinzufügen. Diese Art von Wiedergabeliste ist dynamisch und beinhaltet alle Medien aus der Bibliothek, die gesetzten Regeln erfüllen."
```
@gade this change is wrong. In German it grammatically sometimes correct to have the same word twice in a row. But better style would be: "Bibliothek, welche die"

gade01 replied:
Thanks.
I'll change it.

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (148 of 148 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 66.6% (2 of 3 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 100.0% (83 of 83 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 100.0% (149 of 149 strings)

Added translation using Weblate (Welsh (United Kingdom) (cy_gb))

Added translation using Weblate (Serbian (latin))

Added translation using Weblate (Persian (Iran) (fa_ir))

Added translation using Weblate (Chinese (Taiwan) (zh_tw))

Added translation using Weblate (Chinese (China) (zh_cn))

Added translation using Weblate (Burmese (my_mm))

Added translation using Weblate (Greenlandic (kl_gl))

Added translation using Weblate (Vietnamese (vi_vn))

Added translation using Weblate (Uzbek (uz_uz))

Added translation using Weblate (Albanian (sq_al))

Added translation using Weblate (Dutch (nl_nl))

Added translation using Weblate (Ukrainian (uk_ua))

Added translation using Weblate (Silesian)

Added translation using Weblate (Turkish (tr_tr))

Added translation using Weblate (Thai (th_th))

Added translation using Weblate (Telugu (India) (te_in))

Added translation using Weblate (Tamil (India) (ta_in))

Added translation using Weblate (Tajik (tg_tj))

Translated using Weblate (Danish (da_dk))

Currently translated at 66.6% (2 of 3 strings)

Translated using Weblate (Danish (da_dk))

Currently translated at 66.6% (2 of 3 strings)

Added translation using Weblate (Swedish (sv_se))

Added translation using Weblate (Serbian (sr_rs))

Added translation using Weblate (Persian (Afghanistan) (fa_af))

Added translation using Weblate (Ossetian (os_os))

Added translation using Weblate (Maori)

Added translation using Weblate (Slovenian (sl_si))

Added translation using Weblate (Slovak (sk_sk))

Added translation using Weblate (Sinhala (Sri Lanka) (si_lk))

Added translation using Weblate (Russian (ru_ru))

Added translation using Weblate (Romanian (ro_ro))

Added translation using Weblate (Portuguese (Portugal) (pt_pt))

Added translation using Weblate (Portuguese (Brazil) (pt_br))

Added translation using Weblate (Polish (pl_pl))

Added translation using Weblate (Norwegian Bokmål (nb_no))

Added translation using Weblate (Mongolian (mn_mn))

Added translation using Weblate (Maltese (mt_mt))

Added translation using Weblate (Malayalam (India) (ml_in))

Added translation using Weblate (Malay (ms_my))

Added translation using Weblate (Macedonian (mk_mk))

Added translation using Weblate (Lithuanian (lt_lt))

Added translation using Weblate (Latvian (lv_lv))

Added translation using Weblate (Korean (ko_kr))

Added translation using Weblate (Kannada (India) (kn_in))

Added translation using Weblate (Japanese (ja_jp))

Added translation using Weblate (Italian (it_it))

Added translation using Weblate (Armenian (hy_am))

Added translation using Weblate (Indonesian (id_id))

Added translation using Weblate (Icelandic (is_is))

Added translation using Weblate (Croatian (hr_hr))

Added translation using Weblate (Hungarian (hu_hu))

Added translation using Weblate (Esperanto)

Added translation using Weblate (Hebrew (Israel) (he_il))

Added translation using Weblate (Ukrainian)

Added translation using Weblate (Greek (el_gr))

Added translation using Weblate (Bulgarian (bg_bg))

Added translation using Weblate (Bosnian (Bosnia and Herzegovina) (bs_ba))

Added translation using Weblate (Belarusian (be_by))

Added translation using Weblate (Basque (Spain) (eu_es))

Added translation using Weblate (Azerbaijani (az_az))

Added translation using Weblate (Spanish (Mexico) (es_mx))

Added translation using Weblate (Asturian (Spain) (ast_es))

Added translation using Weblate (Arabic (Saudi Arabia) (ar_sa))

Added translation using Weblate (Spanish (Argentina) (es_ar))

Added translation using Weblate (Spanish (Spain) (es_es))

Added translation using Weblate (Hindi (India) (hi_in))

Added translation using Weblate (German (de_de))

Added translation using Weblate (Galician (Spain) (gl_es))

Added translation using Weblate (French (Canada) (fr_ca))

Added translation using Weblate (French (France) (fr_fr))

Added translation using Weblate (Finnish (fi_fi))

Added translation using Weblate (Czech (cs_cz))

Added translation using Weblate (Catalan (Spain) (ca_es))

Added translation using Weblate (Faroese (fo_fo))

Added translation using Weblate (Afrikaans (South Africa) (af_za))

Added translation using Weblate (Estonian (et_ee))

Added translation using Weblate (English (United States) (en_us))

Added translation using Weblate (Amharic (Ethiopia) (am_et))

Added translation using Weblate (English (New Zealand) (en_nz))

Added translation using Weblate (English (Australia) (en_au))

Added translation using Weblate (Danish (da_dk))

Added translation using Weblate (Welsh (United Kingdom) (cy_gb))

Added translation using Weblate (Serbian (latin))

Added translation using Weblate (Persian (Iran) (fa_ir))

Added translation using Weblate (Chinese (Taiwan) (zh_tw))

Added translation using Weblate (Chinese (China) (zh_cn))

Added translation using Weblate (Burmese (my_mm))

Added translation using Weblate (Greenlandic (kl_gl))

Added translation using Weblate (Vietnamese (vi_vn))

Added translation using Weblate (Uzbek (uz_uz))

Added translation using Weblate (Albanian (sq_al))

Added translation using Weblate (Dutch (nl_nl))

Added translation using Weblate (Ukrainian (uk_ua))

Added translation using Weblate (Silesian)

Added translation using Weblate (Turkish (tr_tr))

Added translation using Weblate (Thai (th_th))

Added translation using Weblate (Telugu (India) (te_in))

Added translation using Weblate (Tamil (India) (ta_in))

Added translation using Weblate (Tajik (tg_tj))

Added translation using Weblate (Swedish (sv_se))

Added translation using Weblate (Serbian (sr_rs))

Added translation using Weblate (Persian (Afghanistan) (fa_af))

Added translation using Weblate (Ossetian (os_os))

Added translation using Weblate (Maori)

Added translation using Weblate (Slovenian (sl_si))

Added translation using Weblate (Slovak (sk_sk))

Added translation using Weblate (Sinhala (Sri Lanka) (si_lk))

Added translation using Weblate (Russian (ru_ru))

Added translation using Weblate (Romanian (ro_ro))

Added translation using Weblate (Portuguese (Portugal) (pt_pt))

Added translation using Weblate (Portuguese (Brazil) (pt_br))

Added translation using Weblate (Polish (pl_pl))

Added translation using Weblate (Norwegian Bokmål (nb_no))

Added translation using Weblate (Mongolian (mn_mn))

Added translation using Weblate (Maltese (mt_mt))

Added translation using Weblate (Malayalam (India) (ml_in))

Added translation using Weblate (Malay (ms_my))

Added translation using Weblate (Macedonian (mk_mk))

Added translation using Weblate (Lithuanian (lt_lt))

Added translation using Weblate (Latvian (lv_lv))

Added translation using Weblate (Korean (ko_kr))

Added translation using Weblate (Kannada (India) (kn_in))

Added translation using Weblate (Japanese (ja_jp))

Added translation using Weblate (Italian (it_it))

Added translation using Weblate (Armenian (hy_am))

Added translation using Weblate (Indonesian (id_id))

Added translation using Weblate (Icelandic (is_is))

Added translation using Weblate (Croatian (hr_hr))

Added translation using Weblate (Hungarian (hu_hu))

Added translation using Weblate (Esperanto)

Added translation using Weblate (Hebrew (Israel) (he_il))

Added translation using Weblate (Catalan (Spain))

Added translation using Weblate (Greek (el_gr))

Added translation using Weblate (Bulgarian (bg_bg))

Added translation using Weblate (Bosnian (Bosnia and Herzegovina) (bs_ba))

Added translation using Weblate (Belarusian (be_by))

Added translation using Weblate (Basque (Spain) (eu_es))

Added translation using Weblate (Azerbaijani (az_az))

Added translation using Weblate (Spanish (Mexico) (es_mx))

Added translation using Weblate (Asturian (Spain) (ast_es))

Added translation using Weblate (Arabic (Saudi Arabia) (ar_sa))

Added translation using Weblate (Spanish (Argentina) (es_ar))

Added translation using Weblate (Spanish (Spain) (es_es))

Added translation using Weblate (Hindi (India) (hi_in))

Added translation using Weblate (German (de_de))

Added translation using Weblate (Galician (Spain) (gl_es))

Added translation using Weblate (French (Canada) (fr_ca))

Added translation using Weblate (French (France) (fr_fr))

Added translation using Weblate (Finnish (fi_fi))

Added translation using Weblate (Czech (cs_cz))

Added translation using Weblate (Catalan (Spain) (ca_es))

Added translation using Weblate (Faroese (fo_fo))

Added translation using Weblate (Afrikaans (South Africa) (af_za))

Added translation using Weblate (Estonian (et_ee))

Added translation using Weblate (English (United States) (en_us))

Added translation using Weblate (Amharic (Ethiopia) (am_et))

Added translation using Weblate (English (New Zealand) (en_nz))

Added translation using Weblate (English (Australia) (en_au))

Added translation using Weblate (Danish (da_dk))

Translated using Weblate (Chinese (Taiwan) (zh_tw))

Currently translated at 100.0% (83 of 83 strings)

Translated using Weblate (Thai (th_th))

Currently translated at 100.0% (79 of 79 strings)

Translated using Weblate (Serbian (latin))

Currently translated at 100.0% (79 of 79 strings)

Translated using Weblate (Serbian (sr_rs))

Currently translated at 100.0% (79 of 79 strings)

Translated using Weblate (Albanian (sq_al))

Currently translated at 100.0% (43 of 43 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 100.0% (80 of 80 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 100.0% (83 of 83 strings)

Translated using Weblate (Macedonian (mk_mk))

Currently translated at 100.0% (49 of 49 strings)

Translated using Weblate (Latvian (lv_lv))

Currently translated at 100.0% (50 of 50 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (83 of 83 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 100.0% (80 of 80 strings)

Translated using Weblate (Hebrew (Israel) (he_il))

Currently translated at 100.0% (80 of 80 strings)

Translated using Weblate (Galician (Spain) (gl_es))

Currently translated at 100.0% (83 of 83 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 100.0% (83 of 83 strings)

Translated using Weblate (French (Canada) (fr_ca))

Currently translated at 100.0% (80 of 80 strings)

Translated using Weblate (Basque (Spain) (eu_es))

Currently translated at 100.0% (79 of 79 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (83 of 83 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 100.0% (80 of 80 strings)

Translated using Weblate (Belarusian (be_by))

Currently translated at 100.0% (80 of 80 strings)

Translated using Weblate (Arabic (Saudi Arabia) (ar_sa))

Currently translated at 100.0% (44 of 44 strings)

Translated using Weblate (Amharic (Ethiopia) (am_et))

Currently translated at 100.0% (59 of 59 strings)

Translated using Weblate (Turkish (tr_tr))

Currently translated at 100.0% (149 of 149 strings)

Translated using Weblate (Thai (th_th))

Currently translated at 100.0% (125 of 125 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 100.0% (148 of 148 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (136 of 136 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (146 of 146 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 100.0% (136 of 136 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (149 of 149 strings)

Translated using Weblate (Japanese (ja_jp))

Currently translated at 100.0% (149 of 149 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 100.0% (149 of 149 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 100.0% (135 of 135 strings)

Translated using Weblate (Hebrew (Israel) (he_il))

Currently translated at 100.0% (135 of 135 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 100.0% (149 of 149 strings)

Translated using Weblate (French (Canada) (fr_ca))

Currently translated at 100.0% (135 of 135 strings)

Translated using Weblate (Basque (Spain) (eu_es))

Currently translated at 100.0% (132 of 132 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 100.0% (149 of 149 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (149 of 149 strings)

Translated using Weblate (Spanish (Argentina) (es_ar))

Currently translated at 100.0% (126 of 126 strings)

Translated using Weblate (Greek (el_gr))

Currently translated at 100.0% (119 of 119 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (148 of 148 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 100.0% (140 of 140 strings)

Translated using Weblate (Bulgarian (bg_bg))

Currently translated at 100.0% (148 of 148 strings)

Translated using Weblate (Amharic (Ethiopia) (am_et))

Currently translated at 100.0% (52 of 52 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 100.0% (3 of 3 strings)

Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Meano Lee <meanocat@gmail.com>
Co-authored-by: vdkbsd <valexgus@gmail.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/am_et/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/ar_sa/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/be_by/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/cs_cz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/eu_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/fr_ca/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/fr_fr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/gl_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/he_il/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/hr_hr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/lv_lv/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/mk_mk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/pt_br/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/ro_ro/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/sq_al/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/sr_Latn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/sr_rs/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/th_th/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/zh_cn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/zh_tw/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/am_et/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/bg_bg/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/cs_cz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/el_gr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/es_ar/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/es_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/et_ee/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/eu_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/fr_ca/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/fr_fr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/he_il/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/hr_hr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/it_it/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/ja_jp/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/nl_nl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/pl_pl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/sk_sk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/th_th/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/tr_tr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/zh_cn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-aac/da_dk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-aac/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-aac/zh_cn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-wma/da_dk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-wma/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-wma/zh_cn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/screensaver-xbmc-builtin-dim/es_mx/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/screensaver-xbmc-builtin-dim/sk_sk/
Translation: Kodi add-ons: skins/skin.estouchy
Translation: Kodi add-ons: skins/skin.estuary
Translation: Kodi core/audioencoder.kodi.builtin.aac
Translation: Kodi core/audioencoder.kodi.builtin.wma
Translation: Kodi core/screensaver.xbmc.builtin.dim
